Source/WebCore/page/LocalDOMWindow.h

```diff
@@ -440,6 +445,16 @@ class LocalDOMWindow final
     // Equivalent to the list of PerformanceEventTiming objects mentioned in https://www.w3.org/TR/event-timing/#sec-modifications-HTML :
     Vector<PerformanceEventTiming::Candidate, 6> m_performanceEventTimingCandidates;
 
+    bool m_contextMenuTriggered = false;
+    // FIXME: support multiple pointers by implementing as a map from Pointer
+    // to PointerState:
+    std::optional<PerformanceEventTiming::InteractionID> m_pointerInteractionId;
+    std::optional<PerformanceEventTiming::Candidate> m_pendingPointerDown;
+
+    HashMap<int, PerformanceEventTiming::Candidate> m_pendingKeyDowns;
+    PerformanceEventTiming::InteractionID m_userInteractionValue = 99;
```

Comment:
I don't think we should be using 99 as a special sentinel value. We should use `std::optional<PerformanceEventTiming::InteractionID>` instead to indicate whether interaction ID has been initialized or not.